[2025-03-21T20:00:17.480+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-03-21T20:00:17.506+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: weather-data-pipeline.create_insert_mart_tables scheduled__2025-03-21T19:00:00+00:00 [queued]>
[2025-03-21T20:00:17.518+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: weather-data-pipeline.create_insert_mart_tables scheduled__2025-03-21T19:00:00+00:00 [queued]>
[2025-03-21T20:00:17.519+0000] {taskinstance.py:2867} INFO - Starting attempt 1 of 2
[2025-03-21T20:00:17.544+0000] {taskinstance.py:2890} INFO - Executing <Task(PythonOperator): create_insert_mart_tables> on 2025-03-21 19:00:00+00:00
[2025-03-21T20:00:17.551+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=2265) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-03-21T20:00:17.553+0000] {standard_task_runner.py:72} INFO - Started process 2267 to run task
[2025-03-21T20:00:17.553+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'weather-data-pipeline', 'create_insert_mart_tables', 'scheduled__2025-03-21T19:00:00+00:00', '--job-id', '163', '--raw', '--subdir', 'DAGS_FOLDER/weather-data-pipeline.py', '--cfg-path', '/tmp/tmpinw0eq36']
[2025-03-21T20:00:17.555+0000] {standard_task_runner.py:105} INFO - Job 163: Subtask create_insert_mart_tables
[2025-03-21T20:00:17.782+0000] {task_command.py:467} INFO - Running <TaskInstance: weather-data-pipeline.create_insert_mart_tables scheduled__2025-03-21T19:00:00+00:00 [running]> on host c25e9da6226b
[2025-03-21T20:00:17.886+0000] {taskinstance.py:3134} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='weather-data-pipeline' AIRFLOW_CTX_TASK_ID='create_insert_mart_tables' AIRFLOW_CTX_EXECUTION_DATE='2025-03-21T19:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-03-21T19:00:00+00:00'
[2025-03-21T20:00:17.887+0000] {taskinstance.py:732} INFO - ::endgroup::
[2025-03-21T20:00:17.906+0000] {base.py:84} INFO - Retrieving connection 'rizqi-neon'
[2025-03-21T20:00:18.481+0000] {python.py:240} INFO - Done. Returned value was: None
[2025-03-21T20:00:18.493+0000] {taskinstance.py:341} INFO - ::group::Post task execution logs
[2025-03-21T20:00:18.493+0000] {taskinstance.py:353} INFO - Marking task as SUCCESS. dag_id=weather-data-pipeline, task_id=create_insert_mart_tables, run_id=scheduled__2025-03-21T19:00:00+00:00, execution_date=20250321T190000, start_date=20250321T200017, end_date=20250321T200018
[2025-03-21T20:00:18.578+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-03-21T20:00:18.762+0000] {taskinstance.py:3901} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-03-21T20:00:18.776+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
